libkbfs/data_types.go

```diff
@@ -36,6 +36,8 @@ type revokedKeyInfo struct {
 
 	// These fields never need copying.
 	sigChainLocation keybase1.SigChainLocation
+	resetSeqno       keybase1.Seqno
+	isReset          bool
```

Comment:
It seems when `isReset` being `false` implies it's a revoke. Not a big deal but if/when you need to change this again, it'd be nice to have a comment here.